Assets/Scripts/Framework/PlayerManagement/PlayerManager.cs

```diff
@@ -116,6 +120,7 @@ public void PlayerJoined(PlayerId playerId)
             }
 
             OnPlayerJoined?.Invoke(playerId);
+            _targetGroup.AddMember(playerId.transform, 1, 1);
```

Comment:
Listen to the `OnPlayerJoined` event in another script instead of declaring Cinemachine code here.

Reply:
Fixed in new branch